as1/src/main/java/socs/network/node/Router.java

```diff
@@ -165,55 +168,134 @@ public void run() {
                 if (inPacket.sospfType == 1) {
                   ports[routerIndex].router2.status = RouterStatus.TWO_WAY;
                   try {
+                    // this point is where the attached router confirms a two way connection
```

Comment:
I don't think this is the right section to add this functionality. Packets 0 1 and 2 should just be used for the initial handshake portion, asking to exchange lsd's should be in a different switch case.

Reply:
The lsd's were updating properly from the protocol ran in start, we need to define a new protocol and packet number to handle more heavyduty swapping.